### DISCUSSION TITLE
Add a simple progress indicator under the cursor for progress bar

### DISCUSSION THREAD

**Discussion by CloneWith** [ppy/osu#30077] [Ideas] [answered]
It would be good to add a progress indicator (similar to the one in the beatmap editor timeline) under the cursor when it lays on the progress bar. The indicator can show the timestamp and progress percentage corresponding to the position of the cursor, which can help users quickly seek an expected point in a replay.

**Comment by Joehuu** [comment, marked as answer]
OP has made PR: https://github.com/ppy/osu/pull/30078
